llvm/test/tools/yaml2obj/GOFF/one-symbol.yaml

```diff
@@ -1,5 +1,7 @@
 # RUN: yaml2obj %s | od -v -An -tx1 | FileCheck --ignore-case %s
 
+## Test for 1 symbol case.
```

Comment:
What coverage does this case give us that e.g. `symbol-all-fields.yaml` doesn't give?